apps/SageAccountWeb/src/components/ConfigureEmail.tsx

```diff
@@ -176,42 +177,42 @@ export const ConfigureEmail = (props: ConfigureEmailProps) => {
           )
         }
       })}
-      <StyledFormControl variant="standard" margin="normal" fullWidth>
-        <Box sx={{ display: 'flex' }}>
-          <TextField
-            label={'Add an email address'}
-            id="additionalEmail"
-            sx={{ flexGrow: 1, marginRight: '10px' }}
-            value={newEmail}
-            onChange={e => setNewEmail(e.target.value)}
-          />
-          <Button
-            startIcon={<AddCircleTwoToneIcon />}
-            disabled={!newEmail}
-            variant={'contained'}
-            sx={{ alignSelf: 'flex-end', height: '47px' }}
-            onClick={e => {
-              addEmail(e)
-            }}
-          >
-            Add
-          </Button>
-        </Box>
-      </StyledFormControl>
-      <FormControlLabel
-        control={
-          <Checkbox
-            checked={sendEmailNotifications}
-            onChange={e => updateEmailNotifications(!sendEmailNotifications)}
-          />
-        }
-        label={
-          <Typography variant="smallText1">
-            Allow Synapse to send me email notifications to my Primary email
-            address.
-          </Typography>
-        }
-      />
+      <Stack my={2} gap={2}>
```

Comment:
Changes to general theme caused this to look funny, so just tweak the layout a bit here

![image](https://github.com/user-attachments/assets/1ad58565-bc46-4612-a917-7528519c91e0)